### DISCUSSION TITLE
BDS?

### DISCUSSION THREAD

**Discussion by FiddlyMC** [DarkWav/SAC-rewrite#10] [Q&A]
When this project is finished will it be compatible with BDS? and if not, do you have any suggestions as to where to find such a thing?

**Comment by radondev** [comment]
This is a pmmp plugin and therefore not compatible with BDS. I doubt it will be finished anytime soon.
As of BDS anticheats, I'm not aware of any solutions.

**Comment by DarkWav** [comment]
This would require a complete re-write in JavaScript and maintainance of two different branches at once, which I currently don't have time for.
Additionally, Microsoft apparently isn't able to provide a consistent and reliable API with deep enough access for an anticheat yet, so I don't know if this is even technically possible at the moment.
